conans/client/tools/scm.py

```diff
@@ -32,7 +32,7 @@ def run(self, command):
         with chdir(self.folder) if self.folder else no_op():
             with environment_append({"LC_ALL": "en_US.UTF-8"}) if self._force_eng else no_op():
                 if not self._runner:
-                    return subprocess.check_output(command, shell=True).decode().strip()
+                    return subprocess.check_output(command, shell=True).decode(errors='ignore').strip()
```

Comment:
@memsharded @lasote :bug: :eyes: Decoding error with non-ascii chars... I come across this issue in SVN because my computer is in spanish and there was an `ó` in one output. Nevertheless the issue could come not only from a hardcoded (or translated) string in Git/SVN but from a filename in the repository. I would suggest to add some non-ascii filenames in tests.

Reply:
I would like to replace all the `decode()` and use the `conans.util.files.decode_text(text)` if there is no other objection.